Sources/MapboxMaps/Gestures/GestureManager+GestureHandlerDelegate.swift

```diff
@@ -12,15 +12,15 @@ extension GestureManager: GestureHandlerDelegate {
 
         // Single tapping twice with one finger will cause the map to zoom in
         if numberOfTaps == 2 && numberOfTouches == 1 {
-            cameraManager.setCamera(to: CameraOptions(zoom: mapView.zoom + 1.0),
+            cameraManager.setCamera(to: CameraOptions(zoom: CGFloat(mapView.cameraState.zoom + 1.0)),
                                     animated: true,
                                     duration: 0.3,
                                     completion: nil)
         }
 
         // Double tapping twice with two fingers will cause the map to zoom out
         if numberOfTaps == 2 && numberOfTouches == 2 {
-            cameraManager.setCamera(to: CameraOptions(zoom: mapView.zoom - 1.0),
+            cameraManager.setCamera(to: CameraOptions(zoom: CGFloat(mapView.cameraState.zoom - 1.0)),
```

Comment:
Where's the mismatch here?

Reply:
ah right.. this was an oversight. Addressed now.